doc/nrf/known_issues.rst

```diff
@@ -979,6 +979,79 @@ KRKNWK-8133: CSMA-CA issues
 SoftDevice Controller
 =====================
 
+.. rst-class:: v1-6-0
+
+DRGN-15817: Possible assert when used together with 802.15.4 protocols when running on nRF53 series.
```

Comment:
@czeslawmakarski: Is this how you want to phrase this?

Reply:
@thomasstenersen I would stress the scanning condition:
`DRGN-15817: Possible assert when used together with 802.15.4 protocols when running on nRF53 series and performing BLE scanning operation.`